scripts/extractions/extract_meteo.py

```diff
@@ -0,0 +1,166 @@
+"""Extract AGERA5 (Meteo) data using OpenEO-GFMAP package."""
+import argparse
+from functools import partial
+from pathlib import Path
+
+import geojson
+import geopandas as gpd
+import openeo
+import pandas as pd
+from extract_sar import (
+    buffer_geometry,
+    create_job_dataframe,
+    filter_extract_true,
+    generate_output_path,
+    pipeline_log,
+    setup_logger,
+    upload_geoparquet_artifactory,
+)
+from openeo_gfmap import Backend, TemporalContext
+from openeo_gfmap.backend import vito_connection
+from openeo_gfmap.manager.job_manager import GFMAPJobManager
+from openeo_gfmap.manager.job_splitters import (
+    append_h3_index,
+    load_s2_grid,
+    split_job_s2grid,
+)
+
+
+def create_datacube_meteo(
+    row: pd.Series,
+    connection: openeo.DataCube,
+    provider=None,
+    connection_provider=None,
+    executor_memory: str = "2G",
+    executor_memory_overhead: str = "1G",
+) -> gpd.GeoDataFrame:
+    start_date = row.start_date
+    end_date = row.end_date
+    temporal_context = TemporalContext(start_date, end_date)
+
+    # Get the feature collection containing the geometry to the job
+    geometry = geojson.loads(row.geometry)
+    assert isinstance(geometry, geojson.FeatureCollection)
+
+    # Filter the geometry to the rows with the extract only flag
+    geometry = filter_extract_true(geometry)
+    assert len(geometry.features) > 0, "No geometries with the extract flag found"
+
+    # Performs a buffer of 64 px around the geometry
+    geometry_df = buffer_geometry(geometry, distance_m=5)
+    spatial_extent_url = upload_geoparquet_artifactory(geometry_df, row.name)
+
+    bands_to_download = ["temperature-mean", "precipitation-flux"]
+
+    cube = connection.load_collection(
+        "AGERA5",
+        temporal_extent=[temporal_context.start_date, temporal_context.end_date],
+        bands=bands_to_download,
+    )
+    filter_geometry = connection.load_url(spatial_extent_url, format="parquet")
+    cube = cube.filter_spatial(filter_geometry)
+    cube.rename_labels(
+        dimension="bands",
+        target=["AGERA5-temperature-mean", "AGERA5-precipitation-flux"],
+        source=["temperature-mean", "precipitation-flux"],
+    )
+
+    # Rescale to uint16, multiplying by 100 first
+    cube = cube * 100
```

Comment:
So it is confirmed that `cube` contains the unscaled values which is why you reintroduce it here?